src/chat-handler.ts

```diff
@@ -38,6 +45,19 @@ export class CodestralHandler extends ChatModel {
       type: 'msg'
     };
     this.messageAdded(msg);
+
+    if (this._llmClient === null) {
+      const botMsg: IChatMessage = {
+        id: UUID.uuid4(),
+        body: '**Chat client not configured**',
```

Comment:
Maybe we can open a new issue to track translating this string (and maybe others) with the `ITranslator`?

Reply:
https://github.com/jupyterlite/jupyterlab-codestral/issues/12